Broken nodejs::npm_from_source in 2.3

Hi, juste updated to 2.3 and the install of npm from source is broken when it does the make install:
```
node: command not found
```

I fixed it for now by installing 'nodejs-legacy'.

I don't know if it's worth it's own bug report, but 'ark' is idempotent on it's source, so because the source were already downloaded, running chef didn't reinstall properly npm. Maybe you could add something like this ? :
```
creates '/usr/local/bin/npm'
```

Thanks.

@guilhem :

```bash
root@npm-source-ubuntu-1204:/usr/local/npm-2.7.4# make install
node cli.js install -g -f 
bash: node: command not found
make: *** [install] Error 127
```

--- a/recipes/npm.rb
+++ b/recipes/npm.rb
@@ -20,7 +20,7 @@
 
 case node['nodejs']['npm']['install_method']
 when 'embedded'
-  include_recipe 'nodejs::nodejs'
+  include_recipe 'nodejs::install'
 when 'source'
   include_recipe 'nodejs::npm_from_source'
 else

--- a/recipes/npm_from_source.rb
+++ b/recipes/npm_from_source.rb
@@ -20,7 +20,7 @@
 
 Chef::Recipe.send(:include, NodeJs::Helper)
 
-include_recipe 'nodejs::nodejs'
+include_recipe 'nodejs::install'
 
 dist = npm_dist
 

